deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -90,20 +90,6 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
-        - name: csi-cephfsplugin-attacher
```

Comment:
please remove the RBAC for this one in the deploy/cephfs folder.

Reply:
@Madhu-1 done. thanks.. ptal.